### ISSUE TITLE
Can this player be installed to WordPress?

### ISSUE THREAD

**Issue by wg1929** [dtube/embed#16]
If you can install to WordPress, we can play ipfs, hope the author can give an answer

**Comment by skzap** [comment]
Yes you can. Just use the iframe code provided on our website like so

```<iframe width="560" height="315" src="https://emb.d.tube/#!/supremegold22/s3k3cjji" frameborder="0" allowfullscreen></iframe>```

**Comment by wg1929** [comment]
@skzap    https://emb.d.tube/    Can I switch to my own player?   http://xxx.com/#!/supremegold22/s3k3cjji

**Comment by SophieElle** [comment]
Hello,
can't get a DTube video to work on my website: copied the iframe-code above and pasted my DTube URL: nothing.

Also downloaded emde master: nothing

Who can help a layman?
By the way: even the Chrome extension does not work for me: nothing is displayed
